packages/venia-ui/lib/RootComponents/Category/category.js

```diff
@@ -123,6 +126,14 @@ Category.defaultProps = {
     pageSize: 6
 };
 
+const mapStateToProps = ({ catalog }) => {
+    const categories = catalog.categories;
```

Comment:
@revanth0212 good catch, I think your suggestion is cleaner and better inline with the 'modulair concept'.

Reply:
I'm not positive about best practices when it comes to nesting connections down. TBH it may not be relevant much longer as we are going to be moving state and actions to context which can be imported/used wherever relevant.

IMO, we already have a connected component here, so there's no need to nest another within in the scope of this PR.

Reply:
Cool then. This comment is moot 👍